src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    /**
+     * main functions like a to-do list
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        String input = "";
+        Boolean end = false;
+        ArrayList<Task> store = new ArrayList(100);
+
+        try {
+            new ReadFile(store);
+        } catch (FileNotFoundException e) {
+            System.err.println(e);
+            System.exit(0);
+        }
+
+        int curr = store.size();
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        // While loop ends when user inputs bye
+        while(!end) {
+            input = scanner.nextLine();
+            String[] splitInput = input.split(" ", 2);
+            try {
+                new DukeException().checker(splitInput, curr);
+            } catch (DukeException e) {
+                System.err.println(e);
+                continue;
+            }
+            // If user inputs bye, terminate the program
+            if (input.equals("bye")) {
+                end = true;
+            }
```

Comment:
Good use of in-line comments, they adhere to the coding standard's requirement as well

Reply:
Agreed! I like how you use in-line comments to help in code readability and understanding the code :)